File: src/main/java/com/paragon/client/systems/module/impl/client/ClientFont.java
package com.paragon.client.systems.module.impl.client;

import com.paragon.api.module.IgnoredByNotifications;
import com.paragon.api.module.Module;
import com.paragon.api.module.Category;

@IgnoredByNotifications
public class ClientFont extends Module {

    public static ClientFont INSTANCE;

    public ClientFont() {
        super("Font", Category.CLIENT, "Use the client's custom font");

        INSTANCE = this;
    }

}

File: src/main/kotlin/com/paragon/impl/module/client/ClientFont.kt
package com.paragon.impl.module.client

import com.paragon.impl.module.Module
import com.paragon.impl.module.Category
import com.paragon.impl.module.annotation.IgnoredByNotifications
import com.paragon.impl.module.annotation.NotVisibleByDefault

/**
 * @author Surge
 */
@IgnoredByNotifications
@NotVisibleByDefault
object ClientFont : Module("Font", Category.CLIENT, "Use the client's custom font")